.github/workflows/ci.yaml

```diff
@@ -46,13 +49,13 @@ jobs:
           NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
         timeout-minutes: 5
 
-      - name: Build versioned docs
-        # TODO support publishing non-dev versions
-        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+      - name: Publish versioned docs
+        if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release/2'))
```

Comment:
we're using master in other repos (default in the org)

Reply:
Time to update, then! Dataplane uses `main` :slightly_smiling_face:

Reply:
Docs are one of the most user-facing repos. For those that are not exposed as well, it's maybe less “important”.

Reply:
It would've been better if it did stick to default so we don't need to switch any :) But I hear you, and we can have that discussion separately. I honestly don't see any reason to bother with it at that stage - in my experience a significant amount (just don't want to say "majority") of OSS projects live completely fine with the `master` branch.

Reply:
They do, they're fine with `master` indeed. I don't see it as a major drawback for the project.

I see changing to `main` as a commitment, though. I do believe that “master” and “slave” have a strong meaning, and should probably never have been used in tech in the first place. But regardless of what I think, there are some people, maybe a small number, to whom the terminology “master”/“slave” brings painful associations, and to whom it matters. They might contribute to the project nonetheless, but not feel as comfortable doing so as they could. By naming our branch differently we acknowledge that this terminology can hurt, and we do our part to keep these words out of the field. More generally, we also indicate that we care about creating an inclusive community. Or at least, we take one step closer to it.